heaps/heap_sort.py

```diff
@@ -1,8 +1,27 @@
-
+from min_heap import MinHeap
+from min_heap import HeapNode
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(n logn)
+        Space Complexity: 0(1)
     """
```

Comment:
👍 This looks right, but since you're using an external heap the space complexity is O(n)